### DISCUSSION TITLE
Error Uno Platform creates dotnet processes without stopping

### DISCUSSION THREAD

**Discussion by tenderodigital** [unoplatform/uno#12775] [Q&A]
Hello:
Since yesterday I can't work with Uno Platform projects in Visual Studio. When I create a project or open an existing one I get this error: it opens dotnet processes without stopping. In the end the PC crashes. I hit out, that's only in 30 seconds.

[INFO] Uno Remote Control initialized (4.9.26+Branch.release-stable-4.9.Sha.10756bf07ebd18380aac88d41128ae4a6f35039c.10756bf07ebd18380aac88d41128ae4a6f35039c)
[INFO] Loaded the Uno.WinUI.RemoteControl Remote Control service (4.9.26).
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[3]
[DEBUG]       Hosting starting
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[4]
[DEBUG]       Hosting started
[DEBUG] dbugHosting environment: Production
[DEBUG] : Microsoft.AspNetCore.Hosting.Diagnostics[13]
[DEBUG]       Loaded hosting startup assembly Uno.UI.RemoteControl.Host
[DEBUG] Content root path: C:\Users\Usuario\.nuget\packages\uno.winui.remotecontrol\4.9.26\tools\rc\host
[DEBUG] Now listening on: http://[::]:56016
[DEBUG] Application started. Press Ctrl+C to shut down.
[INFO] Uno Remote Control initialized (4.9.26+Branch.release-stable-4.9.Sha.10756bf07ebd18380aac88d41128ae4a6f35039c.10756bf07ebd18380aac88d41128ae4a6f35039c)
[INFO] Loaded the Uno.WinUI.RemoteControl Remote Control service (4.9.26).
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[3]
[DEBUG]       Hosting starting
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[4]
[DEBUG]       Hosting started
[DEBUG] Hosting environment: Production
[DEBUG] Content root path: C:\Users\Usuario\.nuget\packages\uno.winui.remotecontrol\4.9.26\tools\rc\host
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[13]
[DEBUG]       Loaded hosting startup assembly Uno.UI.RemoteControl.Host
[DEBUG] Now listening on: http://[::]:56020
[DEBUG] Application started. Press Ctrl+C to shut down.
[INFO] Uno Remote Control initialized (4.9.26+Branch.release-stable-4.9.Sha.10756bf07ebd18380aac88d41128ae4a6f35039c.10756bf07ebd18380aac88d41128ae4a6f35039c)
[INFO] Loaded the Uno.WinUI.RemoteControl Remote Control service (4.9.26).
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[3]
[DEBUG]       Hosting starting
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[4]
[DEBUG]       Hosting started
[DEBUG] Hosting environment: Production
[DEBUG] dbugContent root path: C:\Users\Usuario\.nuget\packages\uno.winui.remotecontrol\4.9.26\tools\rc\host
[DEBUG] Now listening on: http://[::]:56024
[DEBUG] Application started. Press Ctrl+C to shut down.
[DEBUG] : Microsoft.AspNetCore.Hosting.Diagnostics[13]
[DEBUG]       Loaded hosting startup assembly Uno.UI.RemoteControl.Host
[INFO] Uno Remote Control initialized (4.9.26+Branch.release-stable-4.9.Sha.10756bf07ebd18380aac88d41128ae4a6f35039c.10756bf07ebd18380aac88d41128ae4a6f35039c)
[INFO] Loaded the Uno.WinUI.RemoteControl Remote Control service (4.9.26).
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[3]
[DEBUG]       Hosting starting
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[4]
[DEBUG]       Hosting started
[DEBUG] Hosting environment: Production
[DEBUG] Content root path: C:\Users\Usuario\.nuget\packages\uno.winui.remotecontrol\4.9.26\tools\rc\host
[DEBUG] Now listening on: http://[::]:56025
[DEBUG] Application started. Press Ctrl+C to shut down.
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[13]
[DEBUG]       Loaded hosting startup assembly Uno.UI.RemoteControl.Host
[INFO] Uno Remote Control initialized (4.9.26+Branch.release-stable-4.9.Sha.10756bf07ebd18380aac88d41128ae4a6f35039c.10756bf07ebd18380aac88d41128ae4a6f35039c)
[INFO] Loaded the Uno.WinUI.RemoteControl Remote Control service (4.9.26).
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[3]
[DEBUG]       Hosting starting
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[4]
[DEBUG]       Hosting started
[DEBUG] Hosting environment: Production
[DEBUG] dbugContent root path: C:\Users\Usuario\.nuget\packages\uno.winui.remotecontrol\4.9.26\tools\rc\host
[DEBUG] Now listening on: http://[::]:56026
[DEBUG] : Microsoft.AspNetCore.Hosting.Diagnostics[13]
[DEBUG]       Loaded hosting startup assembly Uno.UI.RemoteControl.Host
[DEBUG] Application started. Press Ctrl+C to shut down.
[INFO] Uno Remote Control initialized (4.9.26+Branch.release-stable-4.9.Sha.10756bf07ebd18380aac88d41128ae4a6f35039c.10756bf07ebd18380aac88d41128ae4a6f35039c)
[INFO] Loaded the Uno.WinUI.RemoteControl Remote Control service (4.9.26).
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[3]
[DEBUG]       Hosting starting
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[4]
[DEBUG]       Hosting started
[DEBUG] Hosting environment: Production
[DEBUG] dbugContent root path: C:\Users\Usuario\.nuget\packages\uno.winui.remotecontrol\4.9.26\tools\rc\host
[DEBUG] : Microsoft.AspNetCore.Hosting.Diagnostics[13]
[DEBUG]       Loaded hosting startup assembly Uno.UI.RemoteControl.Host
[DEBUG] Now listening on: http://[::]:56027
[DEBUG] Application started. Press Ctrl+C to shut down.
[INFO] Uno Remote Control initialized (4.9.26+Branch.release-stable-4.9.Sha.10756bf07ebd18380aac88d41128ae4a6f35039c.10756bf07ebd18380aac88d41128ae4a6f35039c)
[INFO] Loaded the Uno.WinUI.RemoteControl Remote Control service (4.9.26).
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[3]
[DEBUG]       Hosting starting
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[4]
[DEBUG]       Hosting started
[DEBUG] dbugHosting environment: Production
[DEBUG] Content root path: C:\Users\Usuario\.nuget\packages\uno.winui.remotecontrol\4.9.26\tools\rc\host
[DEBUG] : Microsoft.AspNetCore.Hosting.Diagnostics[13]
[DEBUG]       Loaded hosting startup assembly Uno.UI.RemoteControl.Host
[DEBUG] Now listening on: http://[::]:56028
[DEBUG] Application started. Press Ctrl+C to shut down.
[INFO] Uno Remote Control initialized (4.9.26+Branch.release-stable-4.9.Sha.10756bf07ebd18380aac88d41128ae4a6f35039c.10756bf07ebd18380aac88d41128ae4a6f35039c)
[INFO] Loaded the Uno.WinUI.RemoteControl Remote Control service (4.9.26).
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[3]
[DEBUG]       Hosting starting
[DEBUG] dbug: Microsoft.AspNetCore.Hosting.Diagnostics[4]
[DEBUG]       Hosting started
[DEBUG] Hosting environment: Production
[DEBUG] Content root path: C:\Users\Usuario\.nuget\packages\uno.winui.remotecontrol\4.9.26\tools\rc\host
[DEBUG] dbugNow listening on: http://[::]:56029
[DEBUG] Application started. Press Ctrl+C to shut down.
[DEBUG] : Microsoft.AspNetCore.Hosting.Diagnostics[13]
[DEBUG]       Loaded hosting startup assembly

I have made the following changes:
1.	Full reinstallation of Visual Studio
2.	Complete reinstallation of Uno Platform. Uno-check does not show any errors.
3.	Reinstalling dotnet.
Can you think of a solution?

**Comment by JesseWBDL** [comment]
I Second that I get a constant
Now listening on: http://[::]xxxxx
this occurs in a massive loop and for every iteration a new .NET Host instance starts until the computer runs out of memory, I had no choice but to uninstall Uno

**Comment by agneszitte** [comment]
Hello @tenderodigital and @JesseWBDL.
Our community reported a bug in the most recent VSIX update on Friday afternoon (Jun 30) and on Sunday (Jul 2) we pushed a fix https://github.com/unoplatform/uno.studio/pull/330. As of Monday (Jul 3) we've updated the extension on marketplace.
We'd appreciate if you can test again with the [latest version **4.9.17.18**](https://marketplace.visualstudio.com/items?itemName=unoplatform.uno-platform-addin-2022&ssr=false#overview) to see if you are still experiencing any troubles.
Thank you in advance for your feedback.

**Comment by tenderodigital** [comment]
Hello, after performing the indicated update, Uno Platform is now working well again in Visual Studio. Thanks a lot.

**Reply by agneszitte** [reply]
Great to hear that @tenderodigital ! Thank you so much for the feedback !
